wiki/Guides/Recording_osu!/en.md

```diff
@@ -1,22 +1,20 @@
 # How to record videos of osu!
 
-[Want to use Virtualdub instead of Sony Vegas? Click here!](https://osu.ppy.sh/forum/viewtopic.php?p=252802#p252802)
+[Want to use Virtualdub instead of Sony Vegas? Click here!](https://osu.ppy.sh/community/forums/topics/18112?start=252802#p252802)
```

Comment:
remove anchor

Reply:
I meant only the part after #, cuz this is supposed to link to the specific post https://osu.ppy.sh/community/forums/topics/18112?start=252802 (or https://osu.ppy.sh/community/forums/posts/252802)